pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -65,9 +67,17 @@ public class MercuryOrderDao implements Dao<MercuryOrderDto> {
             "UPDATE ddp_mercury_sequencing SET order_status = ?, status_date = ?, mercury_pdo_id = ?,  "
                     + " status_detail = ?, status_message = ? where order_id = ?";
 
+    String errorMessage = "Error getting eligible barcodes for a mercury order for participant %s";
```

Comment:
Can you move this error message to the method where it is used? (Looks like it is only used in one place in this file.) When you do that, you can add the DdpParticipantId when it is declared, which makes the code easier to read and lessens the likelihood of mistakes.